test/unit/tasks/seeds_test.rb

```diff
@@ -14,10 +15,16 @@ class SeedsTest < ActiveSupport::TestCase
     Foreman.stubs(:in_rake?).returns(true)
   end
 
-  def seed
+  def seed(*seed_files)
     # Authorisation is disabled usually when run from a rake db:* task
     as_admin do
-      load File.expand_path('../../../db/seeds.rb', __dir__)
+      if seed_files.empty?
+        load File.expand_path('../../../db/seeds.rb', __dir__)
+      else
+        seed_files.each do |file|
+          load File.expand_path("../../../db/seeds.d/#{file}", __dir__)
+        end
+      end
```

Comment:
This could also be written like this, saving a few lines and DRY it up.
```ruby
      seed_files = ['seeds.rb'] if seed_files.empty?
      seed_files.each do |file|
        load File.expand_path("../../../db/seeds.d/#{file}", __dir__)
      end
```